flexget/plugins/operate/digest.py

```diff
@@ -96,10 +96,11 @@ def on_task_learn(self, task, config):
         config = self.prepare_config(config)
         with Session() as session:
             for entry in task.all_entries:
-                if entry.state not in config['state']:
+                state = str(entry.state)
+                if state not in config['state']:
```

Comment:
hmm, maybe a custom json schema format that returns `EntryState`?

Reply:
Not sure what you mean?